test/test_other.py

```diff
@@ -11031,6 +11031,24 @@ def check_symbolmap_info(address, func):
     # The name section will not show bar, as it's inlined into main
     check_symbolmap_info(unreachable_addr, '__original_main')
 
+    # 3. Test symbol map on C++ name mangling
+    self.run_process([EMCC, test_file('other/test_symbolmap.cpp'),
+                      '-O1', '--emit-symbol-map', '-o', 'test_symbolmap.js'])
+    self.assertExists('test_symbolmap.js.symbols')
+
+    out_to_js_call_addr = self.get_instr_addr('call\t0', 'test_symbolmap.wasm')
+    unreachable_addr = self.get_instr_addr('unreachable', 'test_symbolmap.wasm')
+
+    def check_cpp_symbolmap_info(address, func):
```

Comment:
Could we maybe combine this with `check_symbolmap_info` above?

Reply:
We already have a test called `test_symbol_map`  and I just added `other/test_symbol_map.out` which goes with that test.

So calling this `test_symbolmap` is a little confusing.

Maybe move the code from `test_symbol_map` into its own file in `test/other/test_symbol_map.cpp` and then reference that file here?

Reply:
Has moven the two tests into into `test_symbol_map.cpp` file.